books/kestrel/number-theory/tonelli-shanks-proof.lisp

```diff
@@ -1309,7 +1309,6 @@
     (implies (and (natp n)
                   (natp z)
                   (> p 2)
-                  (has-square-root? n p)
```

Comment:
removed this hyp because it can be proved by the has-square-root?-suff theorem in quadratic-residue.lisp